tests/phpunit/tests/test-links-default.php

```diff
@@ -20,10 +20,10 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 	public function set_up() {
 		parent::set_up();
 
+		register_post_type( 'cpt', array( 'public' => true ) ); // translated custom post type
 		self::$model->options['post_types'] = array(
 			'cpt' => 'cpt',
 		);
```

Comment:
```suggestion
		self::$model->options['post_types'] = array( 'cpt' );
```
So it's the same way everywhere.

Reply:
I didn't change it because there loooots of places to modify then. And since it accepts both...